### PR TITLE
feat: expand description box in charts and dashboards

### PR DESCRIPTION
**Pull request by jogeshwar01**
Closes: #7819 

### Description:
Changed TextInput to Textarea to support multiple rows and scroll in our chart and dashboard descriptions.
I have added maxRows of 3 so the box will expand until 3 rows and will scroll from there on.

Example - Update Dashboard
image

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/components/SavedDashboards/AddTilesToDashboardModal.tsx [hunk 1]
```diff
@@ -247,10 +248,13 @@ const AddTilesToDashboardModal: FC<AddTilesToDashboardModalProps> = ({
...
                                 {...form.getInputProps('dashboardName')}
                             />
-                            <TextInput
+                            <Textarea
                                 id="dashboard-description"
                                 label="Dashboard description"
                                 placeholder="A few words to give your team some context"
+                                autosize
+                                maxRows={3}
+                                style={{ overflowY: 'auto' }}
```
Comment (joaoviana) [inline packages/frontend/src/components/SavedDashboards/AddTilesToDashboardModal.tsx:257]: do we need this @jogeshwar01 ?

Comment (jogeshwar01) [reply packages/frontend/src/components/SavedDashboards/AddTilesToDashboardModal.tsx:257]: @joaoviana Its more of a UI/UX thing.
Basically If we skip these, the only thing that's an issue is a default 2 liner textarea even if there is single line of description. Like this :
image

The `autosize` and `maxRows` helps us counter this. We can skip the overflowY:'auto' here.

I would love your opinion on which option we should choose here. Will update the PR accordingly.

Comment (joaoviana) [reply packages/frontend/src/components/SavedDashboards/AddTilesToDashboardModal.tsx:257]: Can you remove the `style={{ overflowY: 'auto' }}` everywhere then @jogeshwar01 ?

**Review by joaoviana — CHANGES_REQUESTED** [review]
Besides the removal of the `overflow` style, one more change is required (since the description can be looooong)

`Dashboard header`'s `Popover.Dropdown` should have `maw={500}`

**Comment by joaoviana** [comment]
just done it! thanks for contributing @jogeshwar01 🙌

**Review by joaoviana — APPROVED** [review]

**Pull request merged**